howso/direct/core.py

```diff
@@ -757,6 +757,78 @@ def get_num_training_cases(self, trainee_id: str) -> Dict:
         """
         return self._execute("get_num_training_cases", {"trainee": trainee_id})
 
+    def get_auto_ablate_params(self, trainee_id: str):
+        """
+        Get parameters set by :meth:`set_auto_ablate_params`.
+        """
+        return self._execute(
+            "set_auto_ablate_params", {"trainee": trainee_id}
+        )
+    
+    def set_auto_ablate_params(
+        self,
+        trainee_id: str,
+        auto_ablate_enabled: bool = False,
+        auto_ablate_weight_feature: str = ".case_weight",
+        minimum_model_size: int = 1_000,
+        influence_weight_entropy_threshold: float = 0.6,
+        exact_prediction_features: Optional[List[str]] = None,
+        residual_prediction_features: Optional[List[str]] = None,
+        tolerance_prediction_threshold_map: Optional[Dict[str, Tuple[float, float]]] = None,
+        relative_prediction_threshold_map: Optional[Dict[str, float]] = None,
+        conviction_lower_threshold: Optional[float] = None,
+        conviction_upper_threshold: Optional[float] = None,
+        **kwargs
```

Comment:
what're the kwargs for?

Reply:
For us to handle experimental arguments that are added without having to make minor changes up the stack for  arguments that may not live on forever